### PR TITLE
Disabling foreign key checks to allow dropping of etl databases and r…

### PR DESCRIPTION
**Pull request by njorocs**
…e-enabling them back

### CODE REVIEW

**Review by patryllus — APPROVED** [review]
Looks okay

File: omod/src/main/resources/sql/DDL.sql [hunk 1]
```diff
@@ -4,7 +4,9 @@ BEGIN
 DECLARE script_id INT(11);
 
 -- create/recreate database kenyaemr_etl
+SET FOREIGN_KEY_CHECKS = 0;
 drop database if exists kenyaemr_etl;
+SET FOREIGN_KEY_CHECKS = 1;
```
Comment (ojwanganto) [inline omod/src/main/resources/sql/DDL.sql:9]: Why not move this at the end of the drop table statements?

File: omod/src/main/resources/sql/dwapi/DDL.sql [hunk 2]
```diff
@@ -4,11 +4,11 @@ BEGIN
 DECLARE script_id INT(11);
 
 -- create/recreate database dwapi_etl
+SET FOREIGN_KEY_CHECKS = 0;
 drop database if exists dwapi_etl;
+SET FOREIGN_KEY_CHECKS = 1;
```
Comment (ojwanganto) [inline omod/src/main/resources/sql/dwapi/DDL.sql:9]: same to this

File: omod/src/main/resources/sql/DDL.sql (hunk 1 above)
Comment (njorocs) [reply omod/src/main/resources/sql/DDL.sql:9]: I noted that by the time we are explicitly dropping each table, we have already dropped the entire database. We also want to keep the check off for the shortest time possible.

**Pull request merged**